ui/app/src/components/HistoryDialog/HistoryDialogContainer.tsx

```diff
@@ -168,7 +162,7 @@ export function HistoryDialogContainer(props: HistoryDialogContainerProps) {
         ])
       );
     } else if (isItemPreviewable) {
-      fetchContentByCommitId(site, item.path, version.versionNumber).subscribe((content) => {
+      fetchContentByCommitId(site, ensureSingleSlash(`/${versionPath}`), version.versionNumber).subscribe((content) => {
```

Comment:
Why would a path not have it's starting slash?
Why is this needed only on this code path?

Reply:
paths are coming without starting slash in the item history API, but only `fetchContentByCommitId` seems to be failing when the path doesn't have its starting slash

Reply:
@sumerjabri @jmendeza is that expected that the paths come without their trailing slash [or is it a bug]?

Reply:
@rart I would say it is a bug, since everywhere we used the path with the leading slash.

Reply:
Fixed here https://github.com/craftercms/studio/pull/3314

Reply:
@jvega190 please update and adjust